recipe/meta.yaml

```diff
@@ -29,6 +29,7 @@ requirements:
     - babel >=1.3
     - docutils >=0.14,<0.18
     - imagesize
+    - importlib-metadata >=4.4
```

Comment:
Do we have `importlib-metadata` on Python 3.10+? What happens when it is installed there? How does this interact with the builtin `importlib.metadata` when available?

Reply:
> Do we have importlib-metadata on Python 3.10+? W

We do, back to e.g. linux-64/importlib-metadata-4.8.1-py310hff52083_1.tar.bz2

> What happens when it is installed there

Nothing too freaky... it doesn't shadow the builtin (`importlib.resources`) or anything.

> interact with the builtin importlib.metadata when available?

In the cases I've found, it looks like it will preferentially import from the [separate package](https://github.com/sphinx-doc/sphinx/blob/f38bd8e9529d50e5cceffe3ca55be4b758529ff7/sphinx/registry.py#L25) before trying the builtin.

Reply:
Interesting would have thought the builtin would be tried first. Have asked about that ( https://github.com/sphinx-doc/sphinx/pull/10007#pullrequestreview-857470666 ).

Reply:
looks like it was answered... again, it's _highly_ likely some package in a good-sized sphinx env is going to have the latest-and-greatest importlib because a lot of authors (pragmatically) can't be bothered to do pins, and i don't blame them. if it _can't_ deal with the backport, then there are bigger problems, and if python keeps changing the API, this problem will basically never go away.

Reply:
Yep agreed. Am use to seeing the other `import` order as the backport package is pretty static and eventually fades out of existence. There are of course exceptions like `mock`. Seems like this is more like `mock` than other backport packages.

If things become more fragmented, like the backport package should be used for certain Python versions and not others, we may need revisit dropping `noarch`.

Though hopefully things settle down eventually.